test/aws-lambda/features/promise-rejection.feature

```diff
@@ -83,5 +83,4 @@ Scenario: promise rejections are reported when using serverless-express
     Then the session is valid for the session reporting API version "1" for the "Bugsnag Node" notifier
     And the session "id" is not null
     And the session "startedAt" is a timestamp
-    And the event "session.events.handled" equals 0
-    And the event "session.events.unhandled" equals 1
+    And the event "session.events" is null
```

Comment:
This promise rejection isn't associated with a session anymore, because the session is started by the Express plugin in the request but an unhandled promise rejection is caught globally and therefore by a different client with no knowledge of the session